src/assets/icons.js

```diff
@@ -157,19 +157,20 @@ const UserIcon = () => {
    );
 };
 
-const CrossIcon = () => {
+const CrossIcon = ({ width, height, handler }) => {
```

Comment:
A better way for more generic usecase,

const CrossIcon = ({ props }) => {

Reply:
Good point, made the changes.